client/utils/utils.go

```diff
@@ -28,6 +27,14 @@ func (gr GasEstimateResponse) String() string {
 	return fmt.Sprintf("gas estimate: %d", gr.GasEstimate)
 }
 
+// MessageOutput respects CLI flags and outputs a message
+func MessageOutput(cliCtx context.CLIContext, txBldr authtxb.TxBuilder, msgs []sdk.Msg, offline bool) error {
```

Comment:
I reckon `MessageOutput` is a misnomer as it does not describe this function's main purpose. What this function does is actually _executing_ the transaction according to the flags set in the context. As a side effect, it prints the response.

Reply:
What would you like this to be called?

Reply:
`HandleTxExecResponse`? Dunno, it's a tricky one, though we'd better get it right now.

CC'ing @cwgoes @alexanderbez

Reply:
going to go with `GenerateOrBroadcastMsgs` per discussions with @alessio and @alexanderbez

Reply:
Sounds good to me!